src/main/java/com/gigacal/controller/CalendarController.java

```diff
@@ -24,9 +20,38 @@ public ResponseEntity<?> getCalendar() {
         return ResponseEntity.status(HttpStatus.OK).body(calendarService.findAll());
     }
 
+    @GetMapping(path = "{id}")
+    public ResponseEntity<?> getCalendarById(@PathVariable("id") Long id) {
+        return ResponseEntity.status(HttpStatus.OK).body(calendarService.findCalendarById(id));
+    }
+
+    @GetMapping(path = "/user/{userId}")
+    public ResponseEntity<?> getCalendarsByUserId(@PathVariable("userId") Long userId) {
+        return ResponseEntity.status(HttpStatus.OK).body(calendarService.findCalendarsByUserId(userId));
+    }
+
+    @GetMapping(path = "/{column}/{parameter}")
+    public ResponseEntity<?> getCalendarsByExactColumn(@PathVariable("column") String column,
```

Comment:
I don't really see the point of adding this general endpoint honestly. Imo it could simply be getCalendarsByName + something to think about - should we allow users to have the more than one calendar with specific name?